tests/enum.lisp

```diff
@@ -229,3 +229,52 @@
       (eval '(defbitfield (bf1 :float)
               (:foo 0))))
   :error)
+
+(defcenum enum.calendar
+  :traditional 
+  (:default :traditional)
+  :gregorian)
+
+(deftest enum.keyword-ref.1
+    (values-list
+     (mapcar (alexandria:curry 'foreign-enum-value 'enum.calendar)
+             '(:traditional :default :gregorian)))
+  0
+  0
+  1)
+
+(defcenum enum.date
+  :era
+  :year 
+  :month 
+  :week_of_year 
+  :week_of_month 
+  :date 
+  :day_of_year 
+  :day_of_week 
+  :day_of_week_in_month
+  :am_pm 
+  :hour 
+  :hour_of_day 
+  :minute 
+  :second 
+  :millisecond 
+  :zone_offset 
+  :dst_offset 
+  :year_woy 
+  :dow_local
+  :extended_year
+  :julian_day
+  :milliseconds_in_day
+  :is_leap_month
+  :field_count
+  (:day_of_month :date))
+
+(deftest enum.keyword-ref.2
```

Comment:
Such a long defcenum example. Is this testing anything different from the previous test?

Reply:
It's just a real world example I had issues with. I'll review those.